<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>noraNormalMapWidget</class>
 <widget class="QDialog" name="noraNormalMapWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Planar intersection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="xRadioButton">
          <property name="text">
           <string>X</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="yRadioButton">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="zRadioButton">
          <property name="text">
           <string>Z</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Resolution: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="resolutionSpinBox">
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>4096</number>
          </property>
          <property name="singleStep">
           <number>4</number>
          </property>
          <property name="value">
           <number>256</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="cacheOutlineButton">
          <property name="text">
           <string>Cache selected mesh's outline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="generateNormlaMapPushButton">
          <property name="text">
           <string>Generate normal map</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>206</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
